news/_posts/2019-09-11-prestashop-in-2019-and-beyond-part-2-pain-points.md

```diff
@@ -427,7 +427,7 @@ For that reason, last year we launched the [DevDocs Project][devdocs], which con
 
 "This has been a very difficult article to write". That's my poor attempt of an excuse for having taken 6 months to put it out the door, and I sincerely apologize for that.
  
-This piece isn't meant to be a "character destruction" for PrestaShop 1.7—quite the opposite. After all, I work for PrestaShop and I'm writing on the official blog, so I'm a little biased. But perfect, pristine, legacy-less software doesn't exist, and it's no news that PrestaShop is not the exception. I think that in the spirit of an open source project, and specially one that aims to become _more_ open, it's important be honest and to tell things as they are. Be proud of our strengths, yes, but also, and above all, acknowledge the things that need improving. Above all, _knowing_ yourself and your weakness is the first step towards doing something meaningful about it.
+This piece isn't meant to be a "character destruction" for PrestaShop 1.7—quite the opposite. After all, I work for PrestaShop and I'm writing on the official blog, so I'm a little biased. But perfect, pristine, legacy-less software doesn't exist, and it's no news that PrestaShop is not the exception. I think that in the spirit of an open source project, and especially one that aims to become _more_ open, it's important be honest and to tell things as they are. Be proud of our strengths, yes, but also, and above all, acknowledge the things that need improving. Above all, _knowing_ yourself and your weakness is the first step towards doing something meaningful about it.
```

Comment:
Today I learnt the difference between specially and especially.

Reply:
It's especially important! :D